wheel/python/clvm_rs/replace.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+from typing import Dict
+
+from .casts import CastableType
+from .clvm_storage import CLVMStorage
+
+
+def replace(program: CLVMStorage, **kwargs: CastableType) -> CastableType:
```

Comment:
I take it you have some future need for this in mind, yeah?

Reply:
This is the implementation for `Program.replace`. The idea was to make `Program` an amalgamation of various disparate api.

This `replace` works on any `CLVMStorage` type. The one in `Program` casts to `Program`, as you'd expect. This follows the philosophy that `Program` can be just a thin wrapper around some other `CLVMStorage` type (like rust's `LazyNode`) and provide it with all the same conveniences.